### ISSUE TITLE
Redirect users to new webmaker.org goggles landing page

### ISSUE THREAD

**Issue by secretrobotron** [mozilla/goggles.webmaker.org#161]
Replace current goggles landing page by the one constructed in #1184 by redirecting to https://webmaker.org/goggles when someone hits https://goggles.webmaker.org/

(Judgement call for @Pomax: assumption is that it's easier to develop this inside of webmaker.org rather than goggles.webmaker.org)

**Comment by Pomax** [comment]
invalid assumption.

Are we integrally switching to "goggles is now a dir on webmaker.org" forever?

**Comment by secretrobotron** [comment]
No, just the landing page. It's up to you where to develop it. @k88hudson had some thoughts about it being easier to develop the landing page inside webmaker.org instead of goggles.webmaker.org.

**Comment by Pomax** [comment]
the process is identical, really. the only difference is the hrefs of the css/js resources.

**Comment by secretrobotron** [comment]
I thought there might be some difference in the way you write the CSS/HTML for that page. Might be useful to just keep it in line with ongoing changes to webmaker.org, as opposed to maintaining static code over here.

**Comment by Pomax** [comment]
all I care about is where this is going to end up being hosted in the end. It makes a difference, but it's mostly an arbitrary one, especially if we're keeping goggles.webmaker.org, so: what are we doing here? goggles.webmaker.org is both a website, and a domain that hosts the goggles JS library. If we want to move the landing page to webmaker.org, that means we stop having goggles.webmaker.org be a website, and only act as JS library host (Otherwise we only doubled our maintenance problems). Doing so will make the issue of goggle's header being out of date a non issue, and that would be nice, but I'd only be up for that if we indeed turn off the "website" part of goggles.webmaker.org =)
